src/init.cpp

```diff
@@ -201,6 +201,7 @@ void DynamicDataLoader::initialize()
     type_function_map["INSTRUMENT"] = new StaticFunctionAccessor(&load_ingored_type);
     // loaded earlier.
     type_function_map["colordef"] = new StaticFunctionAccessor(&load_ingored_type);
+    type_function_map["help_notes"] = new StaticFunctionAccessor( &load_ingored_type );
```

Comment:
I couldn't find any object with type `"help_notes"` (neither existing, nor in this PR). Shouldn't it ignore `"help"` instead?